lib/presentation/demo/demo_screen.dart

```diff
@@ -1,14 +1,16 @@
 import 'package:auto_route/auto_route.dart';
+import 'package:collaction_app/domain/i_settings_repository.dart';
```

Comment:
Prefer relative path imports (we can do this for all imports later, not that important right now)